packages/react/src/components/logged-out/logged-out.test.tsx

```diff
@@ -5,13 +5,15 @@ import {
   TestSlashIDProvider,
   TEST_USER,
 } from "../../context/test-slash-id-provider";
+import { SDKState } from "../../context/slash-id-context";
+import { sdkNotReadyStates } from "../test-utils";
 
 const TestComponent = () => <h1>Test</h1>;
```

Comment:
nit: Maybe use `faker.lorem.sentence()` here too

Reply:
Good idea, let me add it!